apidef/oas/middleware.go

```diff
@@ -475,3 +478,76 @@ type Header struct {
 	Name  string `bson:"name" json:"name"`
 	Value string `bson:"value" json:"value"`
 }
+
+func (ps Paths) fillCache(ep apidef.ExtendedPathsSet) {
+	advanceCacheConfig := &ep.AdvanceCacheConfig
+
+	for _, advCache := range *advanceCacheConfig {
```

Comment:
we can take `cacheMetas []CacheMeta` as param. So we can get rid of 483.
`for _, cm := range cacheMetas {`